swayfmt/src/items/item_storage/mod.rs

```diff
@@ -134,19 +134,10 @@ impl CurlyBrace for ItemStorage {
         line: &mut String,
         formatter: &mut Formatter,
     ) -> Result<(), FormatterError> {
-        let brace_style = formatter.config.items.item_brace_style;
         formatter.indent();
         let open_brace = Delimiter::Brace.as_open_char();
-        match brace_style {
-            ItemBraceStyle::AlwaysNextLine => {
-                // Add opening brace to the next line.
-                write!(line, "\n{open_brace}")?;
-            }
-            _ => {
-                // Add opening brace to the same line
-                write!(line, " {open_brace}")?;
-            }
-        }
+        // Add opening brace to the same line
+        write!(line, " {open_brace}")?;
```

Comment:
Do we have tests for `different brace_style`s?

Reply:
Yeah, the tests are for actual use cases of each type of brace. So for say some array, the test would just be that the array is formatted how you would expect. There isn't any specific test other than that for each implementation of the trait.